### ISSUE TITLE
logo

### ISSUE THREAD

**Issue by bloodchild8906** [bloodchild8906/AUniverseDivided#4]

**Comment by richardbmx** [comment]
@bloodchild8906 Hello my name is Richard I am a graphic designer and I love making contributions for open source projects, I see that you do not have a defined logo for your application. If you want I can do it for you without any problem...

**Comment by bloodchild8906** [comment]
please do. we can see what you come up with

**Comment by bloodchild8906** [comment]
thank you for reaching out if you are interested please do :D look forward to seeing what you come up with

**Comment by richardbmx** [comment]
perfect I will work on this ;-)

**Comment by richardbmx** [comment]
Can you explain to me a little about what the project is about, what is the function and the name of the project

**Comment by bloodchild8906** [comment]
this project is a game. the game is about space exploration and domination basically we want to make a space sim game that allows clans to claim/defend galaxies. 

thats just the initial idea

**Comment by bloodchild8906** [comment]
so there is lots of work for a graphics designer at this point in time. if you are prepared to help out

**Comment by richardbmx** [comment]
perfect we start with the logo for the game and then we see that other things

**Comment by richardbmx** [comment]
@bloodchild8906  I present my first idea of what I hope you like, the design consists of two ships a planet to identify the battle, I present a general logo for the game and an icon version to use in the application

![logo 93](https://user-images.githubusercontent.com/35604837/43175959-9c05f096-8f8f-11e8-91ba-1cc2fdc3c6e6.png)

**Comment by welcomeking** [comment]
hey brother i love your design style we could really use your skill set on ship logos ... for the game as it will not be 2D but 3D game with Hi Resolution ect the current logo which i love really i do i have some illistrator and photoshop background i know time it takes to make these bad boys but id like you to take a look at we have done what or how we plan the game to be and that will give you some context when creating the logo

**Comment by richardbmx** [comment]
perfect that good that you liked the design ...

**Comment by richardbmx** [comment]
Happy day here I leave a Google Drive with all the necessary files for the use of the logo, unfortunately I do not know how to handle it very well github, so I do not know how to assemble and merge the logo, if you want to change the logo or icon you can say it and I will do the change 

All File
[Google Drive](https://drive.google.com/open?id=1ONIXqVV3AJFTDxHdPTaz-DCyxEAL1Haa)

**Comment by richardbmx** [comment]
So you can use the logo without problems

What question do you have?

**Comment by richardbmx** [comment]
@bloodchild8906  @welcomeking  Greetings can place the logo in the readme file. I do not know how to do it so I can begin to give personality to the github game.

**Comment by welcomeking** [comment]
richard whats your email address ?

**Comment by bloodchild8906** [comment]
hey @richardbmx  I would just like to say thank you, however I understand that your english is not great. if we ask something and you do not understand it please ask us to use google translate to get you to understand.

**Comment by richardbmx** [comment]
if I use the google translator but it is not very perfect :-)

**Comment by bloodchild8906** [comment]
thats fine then just ask us to rephrase

**Comment by richardbmx** [comment]
perfect... ;-)

Regarding the logo, do you want any change?

**Comment by bloodchild8906** [comment]
yes please. the logos are good but they are not the style we are looking for, beacuse this game is 3d and not an arcade game. if you can change it please do

we are looking for a more 3d look and not so flat

**Comment by bloodchild8906** [comment]
this is an example of something we are looking for 
![image](https://app.gitkraken.com/api/glo/boards/5b58adfa944d0e0e00a1849d/attachments/5b6504c53aae3f0e00e406e4)
